person.rb

```diff
@@ -1,9 +1,46 @@
-class Person
+require 'securerandom'
+
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'Message'
+  end
+end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
```

Comment:
- [ ] In the Decorator class, please specify the relationship between it and the Nameable class by defining the `attr_accessor` before the `initialize` method 🙏

Reply:
I don't see the need to use an attr_accessor here and I don't see it specified in the requirements either.
What am I supposed to use in the attr_accessor?